File: src/ERP/acme.atena.domain/DTO/Account/PagamentoFormaPagamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.atena.domain.DTO.Account
{
    public class PagamentoFormaPagamento : AbstractEntity
    {
        public PagamentoFormaPagamento()
        {
        }

        public PagamentoFormaPagamento(Guid pagamentoId, Guid formaPagamentoId)
        {
            PagamentoId = pagamentoId;
            FormaPagamentoId = formaPagamentoId;
        }
        public PagamentoFormaPagamento(Guid formaPagamentoId)
        {
            FormaPagamentoId = formaPagamentoId;
        }
        public Guid PagamentoId { get; set; }
        public Guid FormaPagamentoId { get; set; }

        public virtual Pagamento Pagamento { get; set; }
        public virtual FormaPagamento FormaPagamento { get; set; }


    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/account/pagamento-forma-pagamento.ts
import { FormaPagamento } from './forma-pagamento';
import { Pagamento } from './pagamento';
import { AbstractEntity } from "../abstract-entity";

export class PagamentoFormaPagamento extends AbstractEntity{
  public pagamentoId : string;
  public formaPagamentoId: string;

  public pagamento : Pagamento;
  public formaPagamento : FormaPagamento;
}
